index.bs

```diff
@@ -787,9 +799,8 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
     </pre>
 
     <p>
-      The <dfn for='MediaCapabilities' method>decodingInfo()</dfn> method and
-      the <dfn for='MediaCapabilities' method>encodingInfo()</dfn> method MUST
-      run the following steps:
+      The {{decodingInfo()}} method and the {{encodingInfo()}} method MUST run
```

Comment:
Changed this to work around warning for duplicate IDs between here and the IDL definition above. I took from that error that using <dfn...> here was creating 2 definitions. Seems right to just link to the IDL.